src/transformers/generation/utils.py

```diff
@@ -1210,6 +1211,8 @@ def generate(
             streamer (`BaseStreamer`, *optional*):
                 Streamer object that will be used to stream the generated sequences. Generated tokens are passed
                 through `streamer.put(token_ids)` and the streamer is responsible for any further processing.
+            low_memory (`bool`, *optional*):
+                Switch to sequential topk for contrastive search to reduce peak memory requirements.
```

Comment:
```suggestion
                Switch to sequential topk for contrastive search to reduce peak memory requirements. Used with contrastive search.
```

(let's make it clear that it is exclusive to contrastive search)

Reply:
Thanks for adding, very clear